### DISCUSSION TITLE
Exact matching on terms containing hyphens

### DISCUSSION THREAD

**Discussion by amousset** [jameslittle230/stork#311] [Q&A] [answered]
I'm trying to build a search field for https://rustsec.org, and one of the things that would be convenient in this context is the ability to look for various exact ids which contain hyphen (like `CVE-2020-28247`). It looks like stock considers the hyphen as a word delimiter, so these searches do not work.
It doesn't look possible at the moment, but I'm asking just in case if there's a way to make it work.

Anyway, using stork is a pleasure and the docs are particularly good, thanks for this project!

**Comment by jameslittle230** [comment, marked as answer]
@amousset - Thanks for the kind words! You're right that this won't work today since Stork considers hyphens as a word delimiter, and I'm not sure I'd be able to throw together a quick fix for this, but I'm in the middle of a big rewrite of the search algorithm and I'll see if I can add support for this - either a configuration option to no longer use hyphens as a word delimiter, or quoted searches to look for exact matches.

-James
